### DISCUSSION TITLE
dlrs_CaseTrigger error

### DISCUSSION THREAD

**Discussion by MatzJeurissen** [SFDO-Community/declarative-lookup-rollup-summaries#1472] [Q&A]
When trying to merge cases where are experiencing the error attached in the screenshot.
<img width="1178" alt="Screenshot 2024-05-14 at 10 32 57" src="https://github.com/SFDO-Community/declarative-lookup-rollup-summaries/assets/84962321/af50dff8-2aef-4170-accb-bd4ca260b847">

Trailhead issue I've come across:
[Issue 1 ](https://trailhead.salesforce.com/trailblazer-community/feed/0D54S00000HEhupSAD)
[Issue 2](https://trailhead.salesforce.com/trailblazer-community/feed/0D54S00000JeU6rSAF)

We don't have an active lookup summary that aggregrates a child relationship on the case object on the total__c field. Further more there actually isn't a field on the case object called 'total__c'. We do however have other active lookup summaries on different objects that utile a 'total__c' field. As DLRS is a managed package I cannot read the debug logs to figure what is going on and how to resolve it. 

Anyone has any ideas on how to resolve this issue?

**Comment by acrosman** [comment]
From your description it sounds like you have a misconfigured rollup that either targets Case or roles up from Case which uses that `total__c` field.

**Reply by MatzJeurissen** [reply]
Thanks for the response. That was my thought too but when going trough the configurations but couldn't find anything out of the ordinary. I'll go through it again to double check. 

Do you think it could be:

1. A left over field that was added when making a Case configuration => then deleted the configuration => but a bit of the configuration was left behind in the background?
2. Specifically when merging cases something in the logic picks up a field from a different configuration?

**Comment by aheber** [comment]
I don't know the source of this but I've seen it before.

@MatzJeurissen would you be able to create replication steps for this? Making it happen in a clean scratch or dev org?

If you can get the replication steps then we should be able to fix it. I just haven't been able to make it happen in the simplistic versions I've tried in the past. It has been a while since I tried last so maybe you can figure out what mix of settings cause it now?

**Reply by aheber** [reply]
I think this is likely related to #1354, I don't think I've tried to replicate with the details provided on that issue yet. I'll try and schedule some time to look at it but if you beat me to it maybe you can see if that has similarities to your scenario.

**Reply by aheber** [reply]
This should be resolved now, we were able to find and fix a bug that appears to manifest this same way. If someone experiences this in the future please install the latest release and check again. Let us know if that does not resolve the issue.
